Source/SwiftLintCoreMacros/Fold.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftSyntax
+import SwiftSyntaxMacros
+
+struct Fold: ExtensionMacro {
```

Comment:
It'd be cool if you could do

```swift
@Rule
struct MyFirstRule { ... }

@Rule(fold: true)
struct MySecondRule { ... }
```

But I don’t think that’s possible with macros in Swift 5.9.

Reply:
I’m ok being terse with these names because they’re not meant as public API but rather like a DSL for writing SwiftLint rules. Just like `Rule` is pretty generically named.

Reply:
> But I don’t think that’s possible with macros in Swift 5.9.

I cannot try at the moment, but should be possible. It's what I meant with "argument" in the initial comment.

Reply:
Yes if you figure it out that’d be great.

Reply:
#5255 would be my proposal.